doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc

```diff
@@ -3,351 +3,245 @@
-
-Issue: Human task comments are retained and visible only while the respective process instance is running. When the process instance is completed or aborted, comments from human tasks disappear even though the tasks are still visible in {CENTRAL}.
+Issue: If you select multiple tasks on the *Task List* page and then select a bulk reassignment operation, when the dialog box appears over the *Task List* page, the previously selected tasks are no longer selected.
 
 Steps to reproduce:
 
-. Start a process instance with two consecutive human tasks.
-. Comment on the first human task.
-. Claim, start, and complete the first task.
-. Check the *Comments* tab of the human task to confirm that the comments are retained for the completed task.
-. Abort the process instance or complete the task preventing the instance from finishing.
-. Navigate to the log file for the first task again.
+On the *Task List* page, select multiple tasks and choose a bulk reassignment operation.
 
-Expected result: Data about the first task is available in the *Comments* tab of the human task .
+Expected result: A dialog box appears on top of the *Task List* page. The list of tasks is visible on the *Task List* page and the previously selected tasks are still selected.
 
-Actual result: The first task is still available in {CENTRAL} but no data is available in the log file.
+Actual result:  A dialog box appears on top of the *Task List* page. The list of tasks is visible on the *Task List* page and the previously selected tasks are not selected.
 
 Workaround: None.
 
+.The *Bulk Reassign* operation fails if you include tasks that are in a state that does not allow reassignment [https://issues.jboss.org/browse/RHPAM-2386[RHPAM-2386]]
 
-.Greater than (>) and less than (<) symbols in data types break a process [https://issues.jboss.org/browse/RHPAM-2193[RHPAM-2193]]
-
-Issue: If you place greater than (>) and less than (<) symbols in a data type for a process variable,  you cannot reopen the process. The XML editor appears instead.
+Issue: If you select multiple tasks with various statuses, for example *Ready* and *Reserved*, including tasks that are in a state that does not allow reassignment, for example *Suspended* and *Completed*, then the bulk reassign operation fails.
 
 Steps to reproduce:
 
-. Create a process and add the `list1:java.util.List<String>` process variable.
-. Save and reopen the process.
-
-Expected result: The process opens.
+. Select tasks with various statuses and include one in a suspended state.
+. Run the bulk reassignment. An error message appears telling you that some of the the tasks could not be reassigned because they were not in the correct state.
+. Click *OK*.
 
-Actual result: The process does not open.
-
-Workaround: None.
+Expected result: The dialog box closes and tasks are reassigned, except the tasks listed as failed in the dialog box.
```

Comment:
except the tasks listed as failed in the dialog box.

there's no such listing in the dialog. The results (both successful and errors) are presented as notifications in the perspective itself (after the dialog disappeared).

So perhaps:
Expected result: The dialog box closes and operation is attempted on all the tasks. Results for particular tasks are presented separately as notifications.

Reply:
Thanks. Updated.